README.md

```diff
@@ -18,6 +18,8 @@
 
 ## Features
 
+![Searchlib](./docs/volto-searchlib.gif)
```

Comment:
@tedw87 Please use the absolute URL, e.g. https://github.com/eea/volto-redmine-helpdesk/pull/15/files

Reply:
Done